wagtail/search/backends/database/mysql/mysql.py

```diff
@@ -344,9 +345,11 @@ def get_search_field(self, field_lookup, fields=None):
 
     def build_search_query_content(self, query, invert=False):
         if isinstance(query, PlainText):
-            terms = query.query_string.split()
+            # Non-alphanumeric characters act as separators in MySQL full-text search
```

Comment:
Might be worth commenting that this is only relevant for Boolean full text search queries (which is what the Django ORM generates). I plan to look into getting Django to use natural language search mode instead to improve the quality of search results.

Reply:
Sounds fair enough! Comment updated in 573c030037.